publication-commons/src/main/java/no/unit/nva/publication/model/business/DoiRequest.java

```diff
@@ -171,15 +171,12 @@ public void setCustomerId(URI customerId) {
     public void setOwner(User owner) {
         this.owner = owner;
     }
-    
+
     @Override
     public void validateCreationRequirements(Publication publication) throws ConflictException {
         if (publicationDoesNotHaveAnExpectedStatus(publication)) {
             throw new ConflictException(String.format(WRONG_PUBLICATION_STATUS_ERROR, ACCEPTABLE_PUBLICATION_STATUSES));
         }
-        if (publicationHasNvaDoi(publication)) {
-            throw new ConflictException("Publication has NVA issued DOI");
-        }
```

Comment:
Nå opprettes det en DoiRequest ticket for publikasjon med doi, for å kunne gjøre doi findable